models/User.js

```diff
@@ -0,0 +1,34 @@
+/** @module models/User
+* The User Model
+* Schema:
+* username        String      username Required
+* password        String      password Required
+* email						String		  email Required
+* freelancer 			ObjectID 		freelancer profile linked to this user
+*
+* _id (ObjectID) will be added automatically by mongoose if not specified
+*/
+
+'use strict';
+const mongoose = require('mongoose');
+const ObjectID = mongoose.Schema.Types.ObjectId;
+require ('./Freelance');
+
+const User = exports.User = new mongoose.Schema({
+    username    : { type: String, required: true },
+    password    : { type: String, required: true },
+    email				: { type: String, required: true },
+    freelancer	: { type: ObjectID, ref: 'Freelance'},
```

Comment:
Also true.
And even if we put an array, if then we'll have more freelancers the tests will have to be changed, so we can just keep it like this and change everything if we need it later.